sormas-api/src/main/resources/captions.properties

```diff
@@ -244,6 +244,7 @@ bulkDelete=Delete
 bulkDisease=Change Disease
 bulkEdit=Edit...
 bulkEventInvestigationStatus=Change event investigation status
+bulkEventParticipantsToContacts=Generate contacts
```

Comment:
I think I'd prefer "Create contacts" here because "Generate" sounds a bit technical to me - open for discussion though